deployment/params.py

```diff
@@ -1,14 +1,19 @@
-import json
 import typing
 from collections import OrderedDict
 from pathlib import Path
 from typing import Any, List
 
+import yaml
 from ape import chain, project
 from ape.api import AccountAPI, ReceiptAPI
 from ape.cli import get_user_selected_account
 from ape.contracts.base import ContractContainer, ContractInstance, ContractTransactionHandler
 from ape.utils import ZERO_ADDRESS
+from eth_typing import ChecksumAddress
+from ethpm_types import MethodABI
+from web3.auto.gethdev import w3
```

Comment:
Unrelated to this PR, but I'm wondering why we're using gethdev here instead of instantiating a headless `w3 = Web3()` instance. 🤔 Probably just an auto-importing oversight.

Reply:
I grabbed the code from here - https://web3py.readthedocs.io/en/stable/web3.main.html#check-encodability.